csharp_dotnetcore/Using-Cards/Using_Cards/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:3978/",
```

Comment:
This is a different set of ports than DaveTA's samples. Can you set these to match his samples?